.github/workflows/lint.yaml

```diff
@@ -48,5 +48,7 @@ jobs:
     - name: Install revive
```

Comment:
Note, you've removed revive run, but didn't remove the installation step. I put a comment [here](https://github.com/mgechev/revive/pull/1346#issuecomment-2881567252), but it might have got lost. I believe running golangci-lint with the latest bundled revive is not the same as running currently built revive.

Reply:
I agree, it's why I asked what was expected here

https://github.com/mgechev/revive/pull/1346#issuecomment-2880000622